deploy/operator/deploy.sh

```diff
@@ -14,4 +14,15 @@ fi
 
 ${__dir}/setup_lso.sh create_local_volume
 ${__dir}/setup_hive.sh with_olm
+
+default_ocp_versions="${__dir}/../../data/default_ocp_versions.json"
+export OPENSHIFT_VERSIONS=$(cat ${default_ocp_versions} |
```

Comment:
Set `OPENSHIFT_VERSIONS` only if not provided. Otherwise, it won't be possible to override

Reply:
done